src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2208,7 +2208,7 @@
     <value>Edit settings file</value>
   </data>
   <data name="EditSettingsFileDescription" xml:space="preserve">
-    <value>Open settings file in your default editor</value>
+    <value>Open the configuration file in a text editor</value>
```

Comment:
Can you share info on the changed from 'settings' to 'configuration'? I would also avoid the term 'a text editor' considering the file is not a text file.